Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+changeAuthor = env.ghprbPullAuthorLogin ?: CHANGE_AUTHOR
+changeBranch = env.ghprbSourceBranch ?: CHANGE_BRANCH
+changeTarget = env.ghprbTargetBranch ?: CHANGE_TARGET
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && kie-mem16g'
+    }
+    tools {
+        maven 'kie-maven-3.6.3'
+        jdk 'kie-jdk11'
+    }
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '10', numToKeepStr: '')
+        timeout(time: 120, unit: 'MINUTES')
+    }
+    environment {
+        MAVEN_OPTS = '-Xms1024m -Xmx4g'
+    }
+    stages {
+        stage('Initialize') {
+            steps {
+                script {
+                    mailer.buildLogScriptPR()
+
+                    checkoutRepo('optaplanner')
+                    checkoutRepo('optaweb-vehicle-routing')
+                }
+            }
+        }
+        stage('Build OptaPlanner skipping tests') {
+            steps {
+                mavenCleanInstall("optaplanner", true, [])
+            }
+        }
+        stage('Build Optaweb Vehicle Routing') {
+            steps {
+                mavenCleanInstall("optaweb-vehicle-routing", false, ["run-code-coverage", "integration-tests"])
+            }
+        }
+        /* Uncomment to switch SonarCloud analysis from Travis to this pipeline.
+        stage('Analyze Optaweb Vehicle Routing by SonarCloud') {
```

Comment:
@yurloc Please let me know if you want me to switch to SonarCloud analysis here or you prefer to stay with Travis.

Reply:
I would say here. Let's get rid of Travis if it can be replaced without giving up any of its checks. Before we drop Travis, I want to replicate some checks that are done on Travis here. Can you help me with that?

Reply:
If I can I will be happy to help. I would say let's keep this stage commented out before we implement the additional checks in a separate PR, shall we?